install-panel.sh

```diff
@@ -590,16 +590,43 @@ function firewall_firewalld {
   echo "* Firewall-cmd installed"
   print_brake 70
 }
+function centos_based_letsencrypt {
+    if [ "$OS_VER_MAJOR" == "7" ]; then
+      # Enable EPEL folowing official guide https://fedoraproject.org/wiki/EPEL
+      subscription-manager repos --enable "rhel-*-optional-rpms" --enable "rhel-*-extras-rpms"  --enable "rhel-ha-for-rhel-*-server-rpms"
+
+      # Install certbot 
+      yum install certbot
+    elif [ "$OS_VER_MAJOR" == "8" ]; then
+      # Enable EPEL folowing official guide https://fedoraproject.org/wiki/EPEL
+      ARCH=$(/bin/arch)
+      subscription-manager repos --enable "codeready-builder-for-rhel-8-${ARCH}-rpms"
```

Comment:
Using `epel-release` seems fairly straight-forward, isn't this overly complicated? [See this guide.](https://www.digitalocean.com/community/tutorials/how-to-secure-apache-with-let-s-encrypt-on-centos-8#step-1-%E2%80%94-installing-the-certbot-let%E2%80%99s-encrypt-client).

Reply:
@vilhelmprytz I have removed the installation of EPEL from here. The subscription-manager thing is documented in [https://fedoraproject.org/wiki/EPEL](url) and I haven't found it in the script.